### PR TITLE
Fixed preview property attributes

### LINKED ISSUE #830: live preview: property attribute on <img> tag
The live preview for images doesn't work if you add the property attribute to the  tag, only if you add it to a container around the .

greetings, R.H.

### LINKED ISSUE #861: Nested properties not working in preview
If I use a Property in a Property like:

bildschirmfoto 2015-02-23 um 16 59 01

and i change my title. It doesn't update my preview, looks like property="season" blocks property="title"

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #830 , fixes #861
| BC breaks        | TODO
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
Don't you have a PR for sulu-standard applying this as well? And what you have written in the UPGRADE.md file sounds like something that should also go into the documentation.

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js [hunk 1]
```diff
@@ -918,7 +918,7 @@ define([
                 // thats currently not supported by the api
                 var cur = element.parentNode;
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: use `currentNode`, as it is not abbreviated :smile:

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js [hunk 2]
```diff
@@ -918,7 +918,7 @@ define([
...
                 while (null !== cur.parentNode) {
-                    if (cur.hasAttribute('property')) {
+                    if (cur.hasAttribute('property') && cur.getAttribute('typeof') === 'collection') {
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: Do you only want to find blocks here? What's with the other properties?

**Comment by wachterjohannes** [comment]
@danrot we should really document the preview tags (: i will create an issue there! https://github.com/sulu-io/sulu-docs/issues/119

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js (hunk 2 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: when i find a block here i dont want to use this property! thats why i ?`return false` in the next line!

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**